src/reducers/index.js

```diff
@@ -12,9 +12,9 @@ const initialState = {
 export default (state = initialState, action) => {
   switch (action.type) {
     case INCREMENT:
-    // Fill in the body of this case
+    return {count: state.count + 1}
```

Comment:
Good job here. If there were other things in your redux store that you needed to need to preserve while changing your counter, how would you change this?